config/database.json

```diff
@@ -23,7 +23,7 @@
   "development": {
     "username": "postgres",
     "password": "postgres",
-    "database": "fiuba-laboral-v2-backend-database-postgres-development",
+    "database": "development",
```

Comment:
tengo otras bases en la compu. Podemos ponerles: `bolsa-de-trabajo-dev` o `end-of-the-road-dev` o lo que quieras no tan generico